README.md

```diff
@@ -71,6 +194,58 @@ optional arguments:
   -h, --help            show this help message and exit
 ```
 
+### Transmission is called with the following ordered parameters:
+
+```
+<Data Source (ie. "qradar")> <Connection Params: '{"host":"host ip address", "port":"port number", "cert":"certificate"}'> <'{"auth": {authentication}}'> <Transmission Operation: ping, query, status, results or is_async> <Operation input>
+```
+
+**Data source:** This will be the name of the module used for transmission. Currently stix-shifter supports QRadar, Splunk, and Bigfix.
```

Comment:
Translation supports: `QRadar, Splunk, and Elastic`
Transmission supports: `QRadar, Splunk, BigFix`

Making sure this is right

Reply:
Yes that's correct. Bigfix translation is still in progress but its transmission module is in. Elastic isn't an actually data source type so it has no transmission module.